test/Container/ApplicationFactoryTest.php

```diff
@@ -65,6 +65,60 @@ public function getRouterFromApplication(Application $app)
         return $r->getValue($app);
     }
 
+    public function testFactorySetsPhpSettingsFromConfig()
+    {
+        $this->container
+            ->has('Zend\Expressive\Router\RouterInterface')
+            ->willReturn(false);
+        $this->container
```

Comment:
@Maks3w Any idea how to bypass this error when I remove all those expectations:
```
Prophecy\Exception\Call\UnexpectedCallException: Method call:
  - has("Zend\Expressive\Router\RouterInterface")
on Double\ContainerInterface\P65 was not expected,
```
?
It seems like stub yields error when unexpected method call is made.